keyboards/mt/ncr80/hotswap/hotswap.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2022 Jordan Duabe
+ * Copyright 2022 peepeetee
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+
+#define LAYOUT( \
```

Comment:
```suggestion
#define LAYOUT_tkl_ansi_wkl( \
```

Reply:
I don't think this applies, as um the keyboard is missing prtsc, scrlk, and pause

Reply:
Then the info.json is wrong.

Reply:
Noted and corrected

Reply:
These are still outstanding.

Reply:
Could I get some pointers on what I should be doing to the PR at this point? I'm a bit out of the loop and needs some reorientation

Reply:
Look at the reviews.

Reply:
aight done thanks